community/server/src/test/java/org/neo4j/server/rest/transactional/ExecutionResultSerializerTest.java

```diff
@@ -425,27 +432,90 @@ public void shouldSerializePointsAsListOfMapsOfProperties() throws Exception
                       "{\"row\":[{\"type\":\"Point\",\"coordinates\":[12.3,45.6],\"crs\":" +
                         "{\"name\":\"WGS-84\",\"type\":\"link\",\"properties\":" +
                           "{\"href\":\"http://spatialreference.org/ref/epsg/4326/ogcwkt/\",\"type\":\"ogcwkt\"}" +
-                        "}}],\"meta\":[null]}," +
+                        "}}],\"meta\":[{\"type\":\"point\"}]}," +
                       "{\"row\":[{\"type\":\"Point\",\"coordinates\":[123.0,456.0],\"crs\":" +
                         "{\"name\":\"cartesian\",\"type\":\"link\",\"properties\":" +
                           "{\"href\":\"http://spatialreference.org/ref/sr-org/7203/ogcwkt/\",\"type\":\"ogcwkt\"}" +
-                        "}}],\"meta\":[null]}," +
+                        "}}],\"meta\":[{\"type\":\"point\"}]}," +
                       "{\"row\":[{\"type\":\"Point\",\"coordinates\":[12.3,45.6,78.9],\"crs\":" +
                         "{\"name\":\"WGS-84-3D\",\"type\":\"link\",\"properties\":" +
                           "{\"href\":\"http://spatialreference.org/ref/epsg/4979/ogcwkt/\",\"type\":\"ogcwkt\"}" +
-                        "}}],\"meta\":[null]}," +
+                        "}}],\"meta\":[{\"type\":\"point\"}]}," +
                       "{\"row\":[{\"type\":\"Point\",\"coordinates\":[123.0,456.0,789.0],\"crs\":" +
                         "{\"name\":\"cartesian-3D\",\"type\":\"link\",\"properties\":" +
                           "{\"href\":\"http://spatialreference.org/ref/sr-org/9157/ogcwkt/\",\"type\":\"ogcwkt\"}" +
-                        "}}],\"meta\":[null]}," +
-                      "{\"row\":[{\"type\":\"LineString\",\"coordinates\":[[1.0,2.0],[2.0,3.0]],\"crs\":" +
-                        "{\"name\":\"cartesian\",\"type\":\"link\",\"properties\":" +
-                          "{\"href\":\"http://spatialreference.org/ref/sr-org/7203/ogcwkt/\",\"type\":\"ogcwkt\"}" +
-                        "}}],\"meta\":[null]}" +
-                      "]}],\"errors\":[]}",
+                        "}}],\"meta\":[{\"type\":\"point\"}]}" +
+                        "]}],\"errors\":[]}",
                 result );
     }
 
+    @Test
+    public void shouldSerializeTemporalAsListOfMapsOfProperties() throws Exception
+    {
+        // given
+        ByteArrayOutputStream output = new ByteArrayOutputStream();
+        ExecutionResultSerializer serializer = getSerializerWith( output );
+
+        Result executionResult = mockExecutionResult(
+                map( "temporal", LocalDate.of( 2018, 3, 12 ) ),
```

Comment:
Maybe split this test into multiple, like in `TemporalTypeIT`?
Final assertion could then become simpler and not contain full JSON string.

Reply:
Not bother to change this particular test as the string assertion should just capture any format errors.